aten/src/ATen/native/transformers/cuda/flash_attn/fmha_fprop_kernel_1xN.h

```diff
@@ -680,6 +680,11 @@ inline __device__ void device_1xN_loop(const Params &params) {
     // the attention matrix. This way, as long as we have the batch, head, and the location of
     // the 16 x 16 block within the attention matrix, we can generate the exact same dropout pattern.
     auto seeds = at::cuda::philox::unpack(params.philox_args);
+    if (params.philox_args.captured_) {
+        *params.seed = std::get<0>(seeds);
+        *params.extragraph_offset = std::get<1>(seeds);
+    }
+
```

Comment:
I feel like this information is no propgating forward, like the next line should be:
`Philox ph(std::get<0>(seeds), 0, std::get<1>(seeds) + extragraph_offset + (bidb * params.h + bidh) * 32 + tidx % 32);`

But not sure about the first seed